app/uk/gov/hmrc/agentclientrelationships/controllers/Relationships.scala

```diff
@@ -19,42 +19,42 @@ package uk.gov.hmrc.agentclientrelationships.controllers
 import play.api.libs.concurrent.Execution.Implicits.defaultContext
 import play.api.libs.json.Json
 import play.api.mvc.Action
-import uk.gov.hmrc.agentclientrelationships.model.Relationship
+import uk.gov.hmrc.agentclientrelationships.model.{Arn, Relationship}
 import uk.gov.hmrc.agentclientrelationships.repositories.RelationshipRepository
-import uk.gov.hmrc.domain.{AgentCode, SaUtr}
 import uk.gov.hmrc.play.microservice.controller.BaseController
 
 import scala.concurrent.Future
 
 class Relationships(relationshipRepository: RelationshipRepository) extends BaseController {
+    val regime = "mtd-sa"
 
-    def create(saUtr: SaUtr, agentCode: AgentCode) = Action.async {
-        findNonRemoved(saUtr, agentCode).flatMap {
-            case Nil => relationshipRepository.create(saUtr.toString(), "sa", agentCode).map(Json.toJson(_)).map(Ok(_))
+    def create(clientRegimeId: String, arn: Arn) = Action.async {
```

Comment:
Getting the regime from the URL was my plan; I'm being a bit lazy and using the URL spec as a filter for regimes. Given that we've decided to throw away the concept of the regime anyway, it's a moot point.

I'd like to leave renaming clientRegimeId until we remove regimes completely.